lib/NativeCode/include/windows/framework.h

```diff
@@ -11,5 +11,3 @@
 #define STATIC_EXPORT(return_type) extern "C" return_type
 
 #define DLL_EXPORT(return_value) extern "C" __declspec(dllexport) return_value  __stdcall
-
-#define FUN_EXPORT(return_value) DLL_EXPORT(return_value)
```

Comment:
Intentional exclusion? Is this the,

> Remove a duplicate macro definition that was also generating a warning.

referenced in the description?

Reply:
Per your description in the pull request meeting, this is an intentional clean up.